fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1317,6 +1320,7 @@ struct CreateTenantImpl {
 			TenantMapEntry updatedEntry = managementEntry.get();
 			updatedEntry.tenantState = TenantState::READY;
 			ManagementClusterMetadata::tenantMetadata().tenantMap.set(tr, self->tenantName, updatedEntry);
+			ManagementClusterMetadata::tenantMetadata().lastTenantModification.setVersionstamp(tr, Versionstamp(), 0);
```

Comment:
does `0` corresponds to INVALID_VERSION?

Reply:
0 refers to the position in which to insert the Versionstamped value in the value we pass to `setVersionstamp`.  See `set_versionstamped_value ` here: https://apple.github.io/foundationdb/api-python.html. If we use a Versionstamp() class this number will always be 0 since there is no offset to start the value from. If we use something else then this offset might be non-zero